util/api.js

```diff
@@ -3,7 +3,7 @@ import ApiClient from "drp-api-js";
 
 const API_SERVER_BASE =
   (Constants.manifest.env && Constants.manifest.env.EXPO_API_SERVER_BASE) ||
-  "https://icon.doc.ic.ac.uk";
+  "http://10.10.86.75:5000";
```

Comment:
Just a tip, if you need to override the server in development you can use the EXPI_API_SERVER_BASE environment variable.

Reply:
Sorry I'm looking at an old version aren't I

Reply:
Thanks for the tip though